extensions/podman/src/compatibility-mode.ts

```diff
@@ -0,0 +1,121 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import * as extensionApi from '@podman-desktop/api';
+import * as sudo from 'sudo-prompt';
+import * as fs from 'fs';
```

Comment:
```suggestion
import * as fs from 'node:fs';
```

Reply:
then we're sure nobody will fetch https://www.npmjs.com/package/fs :-)

Reply:
thank you! forgot about that :)